docs/docs/guide/google-tag-manager-and-growthbook.mdx

```diff
@@ -0,0 +1,348 @@
+---
+title: GrowthBook and Google Tag Manager
+description: This guide walks you through using GrowthBook with Google Tag Manager (GTM) to track your experiments and measure their impact on your business.
+sidebar_label: Google Tag Manager (GTM)
+slug: google-tag-manager-and-growthbook
+---
+
+# Using GrowthBook with Google Tag Manager (GTM)
+
+Now customers who are familiar with feature management using Google Tag Manager (GTM), yet may lack the engineering resources or capability to implement changes in their codebase, may easily use GrowthBook with GTM to power their AB tests. This setup is commonly used by marketing teams and CRO agencies.
+
+Note: GrowthBook also offers a [visual editor](https://docs.growthbook.io/app/visual) for lightweight, no-code UI changes.
```

Comment:
Not sure if you want to, but we have the 
:::note 
some text
:::
components for adding callouts.

Reply:
TIL, thanks!